tools/ci/build_autoinst.sh

```diff
@@ -8,6 +8,10 @@ set -ex
 destdir=${1:-../os-autoinst}
 sha=${2:-$(cat tools/ci/autoinst.sha)}
 
+[ -d $destdir ] || {
```

Comment:
This lacks quoting but since we use bash just use `[[`:

```suggestion
[[ -d $destdir ]] || {
```